extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -192,11 +179,23 @@ export class AmpAdApiHandler {
    * @private
    */
   updateSize_(height, width, source, origin) {
-    this.baseInstance_.attemptChangeSize(height, width).then(() => {
-      this.sendEmbedSizeResponse_(
-        true /* success */, width, height, source, origin);
-    }, () => this.sendEmbedSizeResponse_(
-        false /* success */, width, height, source, origin));
+    let newHeight, newWidth;
```

Comment:
For 1), `amp-ad` used to have `resizable` attribute, but is removed later. For 2) I can see the height limit definitely doesn't work for `amp-ad`. For 3) I think it makes sense to send response in both cases. 
We need to figure out what are the desired behavior for `amp-ad` and `amp-iframe` resize respectively before we decide if the code should be shared or not.  If we want to keep 1) 2) 3) different, I don't see much point of sharing code.
What's everyone's opinion on this? @cramforce @lannka @muxin